teachers_digital_platform/css/organisms/well.less

```diff
@@ -0,0 +1,36 @@
+// Well styling on Activity search page
+
+.o-well {
+    .tdp-activity-search & {
+        background-color: transparent;
+        max-width: none;
+
+        .h5 {
+            font-size: .875rem;
```

Comment:
Shouldn't this already be 14px or 0.875rem? I don't think you need this.

Reply:
This for an <h4> where styling as .h5.

Reply:
It's a little convoluted, but the `.h5` class already sets `font-size` to 0.875em (14px): [here](https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-base.less#L316-L317) via [here](https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-base.less#L131-L135) via [here](https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-vars.less#L53). Not a big deal and not gonna demand it get changed to merge, though :)